build.gradle.kts

```diff
@@ -71,14 +71,16 @@ kotlin {
         if (chosenTargets.contains("jvm")) {
             val jvmMain by getting {
                 dependencies {
-                    implementation("org.springframework.data:spring-data-r2dbc:3.2.4")
                     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactive:1.8.0")
                     implementation("org.postgresql:r2dbc-postgresql:1.0.4.RELEASE")
                     implementation("io.r2dbc:r2dbc-pool:1.0.1.RELEASE")
                 }
             }
             val jvmTest by getting {
                 dependencies {
+                    // https://mvnrepository.com/artifact/org.slf4j/slf4j-api
+                    implementation("org.slf4j:slf4j-api:2.0.13")
+                    implementation("org.slf4j:slf4j-reload4j:2.0.13")
```

Comment:
I'm not too keen in adding slf4j dependencies, I would rather use https://github.com/oshai/kotlin-logging or similar, and be able to let the user choose which integration they want.

It warrants a separate PR.

Reply:
I totally agree with you on that.
I just used `slf4j` only for the `jvmTest`. I wanted to see the debug logs of the `r2bc` `ConnectionPool` (since is the underlying driver for java targets.
But anyways, If you want I can remove the completely.

Reply:
I'd say either document current usage in README.md or remove it. We'll have to properly think about proper logging in another PR. AFAIK, there's not really a standard in logging for KMP libraries.